composer.json

```diff
@@ -22,7 +22,8 @@
         "twig/twig":                "~1.12"
     },
     "require-dev": {
-      "symfony/expression-language": "~2.4"
+      "symfony/expression-language": "~2.4",
+      "fabpot/php-cs-fixer": "~1.0"
```

Comment:
Adding this requirement looks too much to me. The `php-cs-fixer` should be a globally installed tool (like PHPUnit for example).